ReleaseNotes.md

```diff
@@ -11,6 +11,39 @@ additional questions or comments.
 
 Note : The agent version(s) below has dates (ciprod<mmddyyyy>), which indicate the agent build dates (not release dates)
 
+### 1/31/2022 -
+##### Version microsoft/oms:ciprod01312022 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod01312022 (linux)
+##### Version microsoft/oms:win-ciprod01312022 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-ciprod01312022 (windows)
+##### Code change log
+- Linux Agent
+  - Configurable DB name via configmap for ADX (default DB name:containerinsights)
+  - Default to cAdvisor port to 10250 and container runtime to  Containerd
+  - Update AgentVersion annotation in yamls (omsagent and chart) with released MDSD agent version
+  - Incresing windows agent CPU limits from 200m to 500m
+  - Ignore new disk path that comes from containerd starting with k8s version >= 1.19.x, which was adding unnecessary InsightsMetrics logs and increasing cost
+  - Route the AI SDK logs to log file instead of stdout
+  - Telemetry to collect ContainerLog Records with empty Timestamp
+- Windows Agent
+  - Update to use FluentBit for container log collection and removed FluentD dependency for container log collection
+  - Telemetry to track if any of the variable fields of windows container inventory records has field size >= 64KB
+  - Add windows os check in in_cadvisor_perf plugin to avoid making call in MDSD in MSI auth mode
+  - Bug fix for placeholder_hostname in telegraf metrics
+- Common
+    - Upgrade FluentD gem version from 1.12.2 to 1.14.2
+    - Upgrade Telegraf version from 1.18.0 to 1.20.3
```

Comment:
We could probably add fluent-bit was upgraded from 1.6.8 to 1.7.8 for linux and 1.4.0 to 1.7.8 for windows. Otherwise looks good to me

Reply:
> We could probably add fluent-bit was upgraded from 1.6.8 to 1.7.8 for linux and 1.4.0 to 1.7.8 for windows. Otherwise looks good to me
> thanks for catching this. somehow I missed it. Addressed it now.